docs/UserGuide.md

```diff
@@ -79,13 +79,13 @@ This section introduces ...
 
 ## Commands
 
-This section introduces the various commands used in Nuudle.
+The command is the main way to schedule appointment for a patient with Nuudle.
 
-[general commands](#general-commands), [patient management](#patient-management), [command summary](#command-summary)
+This section provides information about the commands on [general commands](#general-commands), [patient management](#patient-management), [command summary](#command-summary) accordingly.
```

Comment:
This intro feels abit off. Some suggestion that might help:

This section introduces the various commands used in Nuudle

There are three main command types in Nuudle: 
1. General Commands
2. Patient Management Commands
3. Appointment Management Commands

An overview of all Nuudle's commands is also provided in the section below

Reply:
thanks for the suggestion and I did changed it